_together/04-get-involved.md

```diff
@@ -80,7 +80,7 @@ The patterns are a start, but we need your help keeping these patterns current a
 [^2]: Communicating with and about people with disabilities. (February 1, 2022) Retrieved on August 19, 2022, from <https://www.cdc.gov/ncbddd/disabilityandhealth/materials/factsheets/fs-communicating-with-people.html>
 
 ## Credits
-Illustrations created using [humaaans](https://www.humaaans.com/), a [CC0 (public domain)](https://creativecommons.org/share-your-work/public-domain/cc0/) design library.
+Illustrations created using [humaaans](https://www.humaaans.com/), a [CC0 (public domain)](https://creativecommons.org/public-domain/cc0/) design library.
```

Comment:
@amyleadem @mahoneycm - Can you check this link as well just to confirm this is the correct new URL for the intended page? Received a new test failure for this link.

Reply:
> [!note]
> Confirming content is the same as the old url. [Before](https://web.archive.org/web/20230302144541/https://creativecommons.org/share-your-work/public-domain/cc0/) | [After](https://creativecommons.org/public-domain/cc0/)

Reply:
Lgtm!